rdkit/Chem/AtomPairs/Torsions.py

```diff
@@ -54,33 +54,27 @@ def pyScorePath(mol, path, size, atomCodes=None):
   >>> c3 = Utils.GetAtomCode(m.GetAtomWithIdx(2),2)
   >>> c4 = Utils.GetAtomCode(m.GetAtomWithIdx(4),1)
   >>> t = c1 | (c2 << rdMolDescriptors.AtomPairsParameters.codeSize) | (c3 << (rdMolDescriptors.AtomPairsParameters.codeSize*2)) | (c4 << (rdMolDescriptors.AtomPairsParameters.codeSize*3))
-  >>> pyScorePath(m,(0,1,2,4),4)==t
+  >>> pyScorePath(m, (0, 1, 2, 4), 4) == t
   1
 
   """
   codes = [None] * size
   for i in range(size):
-    if i == 0 or i == (size - 1):
-      sub = 1
-    else:
-      sub = 2
+    sub = 1 if i in (0, size - 1) else 2
     if not atomCodes:
       codes[i] = Utils.GetAtomCode(mol.GetAtomWithIdx(path[i]), sub)
     else:
-      base = atomCodes[path[i]]
-      codes[i] = base - sub
+      codes[i] = atomCodes[path[i]] - sub
 
   # "canonicalize" the code vector:
-  beg = 0
-  end = len(codes) - 1
-  while (beg < end):
-    if codes[beg] > codes[end]:
-      codes.reverse()
-      break
-    elif codes[beg] == codes[end]:
+  beg, end = 0, len(codes) - 1
```

Comment:
I'm not a fan of converting multiple assignments into a single one using tuple unpacking. 

My rationale for this is that it makes reading the code harder as it requires extra attention to associate the individual parts with each other. Looking at the old code, this is immediately clear. 

Code readability is key for a shared code base and I therefore discourage the use of this. @greglandrum, as the owner of the code base, your decision.

Reply:
I agree with @gedeck.
I think assigning from a tuple directly is great, something like:
```
a, b = function_returning_a_tuple()
```
but the form you have here is difficult to read. I'd prefer:
```
beg = 0
end = len(codes) - 1
```